packages/server/lib/server-e2e.ts

```diff
@@ -53,42 +49,8 @@ export class ServerE2E extends ServerBase<SocketE2E> {
     this._urlResolver = null
   }
 
-  open (config: Record<string, any> = {}, project, onError, onWarning, shouldCorrelatePreRequests) {
-    debug('server open')
-
-    la(_.isPlainObject(config), 'expected plain config object', config)
-
-    return Bluebird.try(() => {
-      const app = this.createExpressApp(config)
-
-      logger.setSettings(config)
-
-      this._nodeProxy = httpProxy.createProxyServer()
-      this._socket = new SocketE2E(config)
-
-      const getRemoteState = () => {
-        return this._getRemoteState()
-      }
-
-      this.createNetworkProxy(config, getRemoteState, shouldCorrelatePreRequests)
```

Comment:
If you are wondering where big chunk of code went, it's in `server-base.ts` now: https://github.com/cypress-io/cypress/pull/16981/files#diff-040b9d41ce1efd9534f52c69af25d3d587c3a5b9cc9cb8b39a2848ec3e7e420eR204